src/xercesc/util/XMLChar.cpp

```diff
@@ -8837,7 +8837,7 @@ XMLByte XMLChar1_1::fgCharCharsTable1_1[0x10000] =
 
 #include <stdio.h>
 
-static XMLCh gTmpCharTable[0xFFFF];
```

Comment:
I'm unsure about these any other code changes, above.  Are they intentional?

If they are, please could you open separate PRs for them so we can keep code fixes separate from the fuzzer addition.